gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -114,20 +114,17 @@ void GazeboRosCamera::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPtr _
   // Camera info publisher
   // TODO(louise) Migrate ImageConnect logic once SubscriberStatusCallback is ported to ROS2
   impl_->camera_info_pub_ = impl_->ros_node_->create_publisher<sensor_msgs::msg::CameraInfo>(
-    impl_->camera_name_ + "/camera_info");
+    impl_->camera_name_ + "/camera_info", rclcpp::SensorDataQoS());
```

Comment:
I noticed that the camera info reliability setting is "best effort" but the image topic is "reliable". I think the image topic should be updated too. See patch here: https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1031